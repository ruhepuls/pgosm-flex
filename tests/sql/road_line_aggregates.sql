SELECT COUNT(*),
        COUNT(*) FILTER (WHERE maxspeed IS NOT NULL),
        COUNT(*) FILTER (WHERE major),
        COUNT(*) FILTER (WHERE route_foot),
        COUNT(*) FILTER (WHERE route_cycle),
        COUNT(*) FILTER (WHERE route_motor)
    FROM osm.road_line
;
