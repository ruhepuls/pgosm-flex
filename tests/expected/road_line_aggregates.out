34645|1139|6880|26075|22080|24424
